cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -76,6 +77,7 @@ type Planner struct {
 	cc                    controllerReplicasCalculator
 	scaleDownSetProcessor nodes.ScaleDownSetProcessor
 	scaleDownContext      *nodes.ScaleDownContext
+	longestNodeScaleDownT *longestNodeScaleDownEvalTime
```

Comment:
nit: let's pay the tax of the additional 3 characters and call this `longestNodeScaleDownTime`